src/pages/index.js

```diff
@@ -45,6 +51,10 @@ buttonAdd.addEventListener("click", function (evt) {
   addCardPopup.open();
 });
 
+avatarContainerElement.addEventListener("click", () => {
+  editAvatarPopUp.open();
+});
+
 // form validator
 
 const options = {
```

Comment:
From the 8th sprint all constants should be in file `constants.js` in folder `utils` to clean up the main file.